utils/python/CIME/test_status.py

```diff
@@ -18,21 +18,23 @@
 # Special statuses that the overall test can be in
 TEST_DIFF_STATUS     = "DIFF"   # Implies a failure in one of the COMPARE phases
 NAMELIST_FAIL_STATUS = "NLFAIL" # Implies a failure in the NLCOMP phase
+TEST_BFAIL_STATUS    = "BFAIL"  # Implies baselines are missing in the COMPARE_baseline phase
```

Comment:
Based on @jgfouca 's comment: It seems like there shouldn't be a separate overall status for BFAIL. However, I'd still support having "BFAIL" be defined somewhere in this file as a special string that can appear in comments, to clearly signal that this string has some special meaning.

Maybe you can introduce a new section of constants here like this:

```python
# Special strings that can appear in comments, indicating particular types of failures
TEST_NO_BASELINES_COMMENT = "BFAIL" # Implies baseline directory is missing in the baseline comparison phase
```